uptycs/models.go

```diff
@@ -298,6 +298,28 @@ type Roles struct {
 	Limit  int        `json:"limit,omitempty"`
 }
 
+type ComplianceProfile struct {
+	ID                            string        `json:"id"`
+	Name                          string        `json:"name,omitempty"`
+	Description                   string        `json:"description,omitempty"`
+	Custom                        bool          `json:"custom"`
+	Priority                      int           `json:"priority,omitempty"`
+	CreatedBy                     string        `json:"createdBy"`
+	UpdatedBy                     string        `json:"updatedBy"`
+	CreatedAt                     string        `json:"createdAt"`
+	UpdatedAt                     string        `json:"updatedAt"`
+	Links                         []LinkItem    `json:"links"`
+	ComplianceProfileObjectGroups []ObjectGroup `json:"complianceProfileObjectGroups"`
+}
+
+type ComplianceProfiles struct {
+	Links      []LinkItem          `json:"links,omitempty"`
+	Items      []ComplianceProfile `json:"items,omitempty"`
+	Offset     int                 `json:"offset,omitempty"`
+	Limit      int                 `json:"limit,omitempty"`
+	Decorators []string            `json:"decorators,omitempty"`
```

Comment:
Weird, this is unique to complianceProfiles and might be a bug. ask if this is actually real in the uptycs channel perhaps?

Reply:
Messaged uptycs